CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## [2.4.2](https://github.com/IBM/kubernetes-driver/tree/2.4.2) (2023-11-13)
```

Comment:
Please don't change anything in the file CHANGELOG.md. This file will be changed during release of the driver with all the fixes that the release will have.

Reply:
Yeah sure, I've just reverted the change ..